packages/foundation/tests/shadows.chroma.jsx

```diff
@@ -10,7 +10,7 @@ stories()
     .add("default",
          () =>
              <Inline gap={12}>
-                 <div className="w13 h13 shadow-100 mb8" />
-                 <div className="w13 h13 shadow-200" />
+                 <div className="w13 h13 shadow-100 bg-white mb8" />
```

Comment:
Why? If you need to set a background-color shouldn't it use an indirection?

Reply:
These indirection classes are not created. If we expose these classes it makes sense to use `foreground-color-1` instead.

Reply:
As discussed in a call yesterday, we should create them!

Reply:
Where would you define them? In Tachyons? If so any name idea, Alias?

Reply:
done, might need to be moved and expand upon later on after discussions with designers